src/services/chatgpt.ts

```diff
@@ -0,0 +1,88 @@
+const BASE_URL = 'https://api.openai.com/v1'
+const apiKey = '11111'
+
+interface EventDetails {
+  summary: string
+  location: string
+  start: {
+    dateTime: Date
+    timeZone: string
+  }
+  end: {
+    dateTime: Date
+    timeZone: string
+  }
+  description: string
+}
+
+type GetEventDetailsSuccessResponse = {
+  id: string
+  object: string
```

Comment:
Remove unnecessary properties from the response types, like this `object` field.